DuckDuckGo/Common/Extensions/URLExtension.swift

```diff
@@ -347,6 +347,10 @@ extension URL {
         return ![.https, .http, .about, .file, .blob, .data, .ftp, .javascript, .duck].contains(navigationalScheme)
     }
 
+    var isWebExtensionUrl: Bool {
+        return scheme == "webkit-extension"
```

Comment:
let‘s add the scheme as a `NavigationExtension` case and check for `url.navigationScheme == .webkitExtension` to keep the same style